.github/workflows/bsd.yaml

```diff
@@ -3,12 +3,12 @@ on: [push, pull_request]
 
 jobs:
   freebsd:
-    runs-on: macos-12
+    runs-on: ubuntu-22.04
```

Comment:
is this correct? I thought FreeBSD only used to work on the OSX builders; is there a doc reference to talk about it?